native-apps/unity_ii_universallink/ii_integration_dapp/src/greet_frontend/src/index.js

```diff
@@ -0,0 +1,95 @@
+import {createActor, greet_backend} from "../../declarations/greet_backend";
+import {AuthClient} from "@dfinity/auth-client";
+import {HttpAgent} from "@dfinity/agent";
+import {DelegationIdentity, Ed25519PublicKey, ECDSAKeyIdentity, DelegationChain} from "@dfinity/identity";
+import {fromHexString} from "@dfinity/identity/lib/cjs/buffer";
+
+let appPublicKey;
+
+var url = window.location.href;
+var publicKeyIndex = url.indexOf("sessionkey=");
+if (publicKeyIndex !== -1) {
+    // Parse the public key.
+    var publicKeyString = url.substring(publicKeyIndex + "sessionkey=".length);
+    appPublicKey = Ed25519PublicKey.fromDer(fromHexString(publicKeyString));
+}
+
+let actor = greet_backend;
+let delegationChain;
+
+const loginButton = document.getElementById("login");
+loginButton.onclick = async (e) => {
+    e.preventDefault();
+
+    // Create an auth client.
+    var middleKeyIdentity = await ECDSAKeyIdentity.generate();
```

Comment:
Is it documented somewhere why this `middleKeyIdentity` is needed? This is not obvious for somebody new to the platform looking for guidance.

Reply:
Fixed at https://github.com/dfinity/examples/pull/734/commits/63209beee1eacf11c9c3af23984913fe87e8110c